layouts/partials/home/cncf-status.html

```diff
@@ -0,0 +1,17 @@
+<section class="container-xl">
+  <div class="row">
+    <div class="col-12">
+      <hr>
+      <p>
+        <a href="https://cncf.io">
+          <img src="img/logos/cncf-color.png" width="200">
+        </a>
+        </br>
+        We are a Cloud Native Computing Foundation sandbox project.
```

Comment:
Or whatever the shortcode happens to be.
```suggestion
        {{ .project }} is a Cloud Native Computing Foundation sandbox project.
```

Reply:
I like this suggestion, I'll build out a shortcode.